modules/auth/views/Login.vue

```
@@ -93,12 +93,11 @@
 <script setup lang="ts">
 import { ref } from 'vue'
 import { useRules } from '~/composables/useRules'
-import { useAuthWrapper } from '../composables/useAuthWrapper'
 import { useAuthStore } from '../store/auth.store'
 
 const router = useRouter()
-const { isLoading } = useAuthStore()
-const { login } = useAuthWrapper()
+const authStore = useAuthStore()
+const { isLoading } = storeToRefs(authStore)
```

ESLint

[error] 'storeToRefs' is not defined.

(no-undef)